Base/res/js/Spreadsheet/runtime.js

```diff
@@ -197,6 +197,18 @@ class CommonRange {
         });
         return new SplitRange(cellNames);
     }
+
+    unique() {
+        const cellNames = [];
+        const values = new Set();
+        this.forEach(cell => {
+            if (!values.has(cell.value())) {
+                values.add(cell.value());
+                cellNames.push(cell.name);
```

Comment:
Can we make it so `SplitRange` can take an Array of Positions as well, so we don't have to parse the cell name every time?

Reply:
done :)